web/src/js/ducks/ui/index.ts

```diff
@@ -5,11 +5,31 @@ import optionsEditor from "./optionsEditor";
 import tabs from "./tabs";
 import filter from "./filter";
 
-// TODO: Just move ducks/ui/* into ducks/?
+const initialPreferencesState = {
+    timezoneDisplay: "utc" as "utc" | "local",
```

Comment:
I'd default to local time, because I don't think most people want to see UTC. It feels more natural to work in local time.